wpeverest/everest-forms
# Enhancement - New add-ons listing in add-ons Section.

### All Submissions:






### Changes proposed in this Pull Request:



New add-ons from Calculations to Pipedrive listed on add-ons page.

### How to test the changes in this Pull Request:

Verify, whether new add-ons from Calculations to Pipedrive are listed or not on the add-ons page. And also verify, whether the installation/activation/deactivation of these add-ons works properly or not.
### Types of changes:




### Other information:




### Changelog entry

> Enhancement - New add-ons from Calculations to Pipedrive listing in add-ons page.
